ruby-challenges.rb

```diff
@@ -10,7 +10,16 @@
 num3 = 221
 # Expected output: '221 is odd'
 
-
+def even_odd number
+    if number % 2 == 0
```

Comment:
Try playing around with some of the Ruby built-in methods available to you.  There are some methods that help us check for even or odd (Hint: .even? or .odd?)